docs/pages/interfaces.rst

```diff
@@ -3,8 +3,101 @@
 Interfaces
 ==========
 
-On inheritance
---------------
+General Information
+-------------------
+
+All the non-specific interfaces (e.g. MappableN, BindableN, ApplicativeN) can
+have **Nth** types, at the maximum of three possible types. What does this mean?
+
+:class:`MappableN <returns.interfaces.mappable.MappableN>` interface,
+for example, can have one, two or three possible types. See the example below:
+
+.. code:: python
+
+  >>> from typing import NoReturn
+
+  >>> from returns.interfaces.mappable import MappableN, Mappable1, Mappable2, Mappable3
+
+  >>> one_type: MappableN[int, NoReturn, NoReturn]
+  >>> two_types: MappableN[int, str, NoReturn]
+  >>> three_types: MappableN[int, str, bool]
+  >>> # We have a shortcut for each amount of arguments to reduce the boilerplate
+  >>> one_type: Mappable1[int]
+  >>> two_types: Mappable2[int, str]
+  >>> three_type: Mappable3[int, str, bool]
+
+.. note::
+
+  Useful links before you start here:
+
+  * `Functors, Applicatives, And Monads In Pictures <http://adit.io/posts/2013-04-17-functors,_applicatives,_and_monads_in_pictures.html>`_
```

Comment:
Let's replace "functor", "monad" etc with our own names: mappable, bindable, etc.
See #451

Reply:
`Functors, Applicatives, And Monads In Pictures` is the article's name, can be a little weird change the name to `Mappable, Applicative, Bindable in Pictures`. The user can be confused hahaha

But in the other scenarios, I agree with you!

Reply:
Oh, sorry. Yes, articles can be named whatever they originally have in their title.

Reply:
Done!